### PR TITLE
Removed AnnotatedType check from AnnotationUtils

### PR DESCRIPTION
**Pull request by lefloh**

### CODE REVIEW

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java [hunk 1]
```diff
@@ -65,25 +67,33 @@
 public class AnnotationUtilsTest {
 
 	@Inject
-	private SomeController someController;
-	
+	private SomeController someController; // not proxied
```
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: Don't use trailing comments\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java [hunk 2]
```diff
@@ -65,25 +67,33 @@
...
+
 	@Inject
-	private SomeBean someBean;
+	private SomeBean someBean; // proxied
```
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: Don't use trailing comments\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java [hunk 3]
```diff
@@ -100,20 +119,50 @@ public void hasAnnotationOnMethod() throws NoSuchMethodException {
...
-		@NotNull(message = "notNull") public void notNull() {}
+	static class SomeBean {
+		@NotNull(message = "notNull") public String notNull() {
+            return "something";
+        }
 	}
 
+    @Controller
+    private static class BaseController {
+        @View("start-base.jsp") @Path("/base") public void start() {}
```
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\{' is not followed by whitespace\.

Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\}' is not preceded with whitespace\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java [hunk 4]
```diff
@@ -100,20 +119,50 @@ public void hasAnnotationOnMethod() throws NoSuchMethodException {
...
+    }
+
+    private static class InheritedController extends BaseController {}
```
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\{' is not followed by whitespace\.

Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\}' is not preceded with whitespace\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java [hunk 5]
```diff
@@ -100,20 +119,50 @@ public void hasAnnotationOnMethod() throws NoSuchMethodException {
...
+
+    @Controller
+    private interface ControllerInterface {
+        @View("start-interface.jsp") void start();
+    }
+
+    private static class ControllerImpl implements ControllerInterface {
+        public void start() {}
```
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\{' is not followed by whitespace\.

Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\}' is not preceded with whitespace\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java [hunk 6]
```diff
@@ -100,20 +119,50 @@ public void hasAnnotationOnMethod() throws NoSuchMethodException {
...
+    }
+
+    private static class InheritedControllerImpl extends BaseController implements ControllerInterface {}
```
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\{' is not followed by whitespace\.

Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\}' is not preceded with whitespace\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java [hunk 7]
```diff
@@ -100,20 +119,50 @@ public void hasAnnotationOnMethod() throws NoSuchMethodException {
...
+
+    private static class NoInheritanceController extends BaseController {
+        @View("start.jsp") public void start() {}
```
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\{' is not followed by whitespace\.

Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\}' is not preceded with whitespace\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java (hunk 1 above)
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: Don't use trailing comments\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java (hunk 2 above)
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: Don't use trailing comments\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java (hunk 3 above)
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\{' is not followed by whitespace\.

Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\}' is not preceded with whitespace\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java (hunk 4 above)
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\{' is not followed by whitespace\.

Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\}' is not preceded with whitespace\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java (hunk 5 above)
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\{' is not followed by whitespace\.

Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\}' is not preceded with whitespace\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java (hunk 6 above)
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\{' is not followed by whitespace\.

Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\}' is not preceded with whitespace\.

File: ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java (hunk 7 above)
Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\{' is not followed by whitespace\.

Comment (sputnikci) [inline ozark/src/test/java/org/glassfish/ozark/util/AnnotationUtilsTest.java]: \[Checkstyle\] ERROR: '\}' is not preceded with whitespace\.

**Comment by chkal** [comment]
Looks good to me. IMO the checkstyle violations are false positives. Perhaps we should fine tune the rules.

**Pull request merged**